### PR TITLE
Document service base

### PR DESCRIPTION
**Pull request by duranmla**
**What:** Add base to develop the document generation service

**Why:** related to #36 

**How:**
- Add base resources to service using AWS Cloud Formation throughout serverless framework
    - SQS
    - S3 Bucket
- Add babel to allow familiar syntaxis while developing service
- Add flow support and eslint
- Add basic logic to receive a message and create a PDF from the received message
- Upload generated PDF to declared Bucket
- Add custom package building
- Solve chrome runner instance for lambda
- Fix node version to avoid issues _(we enforce nodejs8.10)_ due to puppeteer + chrome lambda requirements

_Note: the video shows manual navigation through the console in order to see the file since the ACL rule is not set to "public-read"_

_Lastly, the background of final decision for implementation can be found at https://github.com/debtcollective/disputes/issues/16#issuecomment-500985203_

http://recordit.co/M6O2HBub4y

Next steps commented at https://github.com/debtcollective/disputes/issues/36#issuecomment-505527142

### CODE REVIEW

File: apps/document-generation/.serverless/cloudformation-template-create-stack.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (orlando) [inline apps/document-generation/.serverless/cloudformation-template-create-stack.json]: We can ignore the `.serverless` directory. Let's add it to `.gitignore`

File: apps/document-generation/src/templates/credit_report_dispute_letter/0.pug [hunk 2]
```diff
@@ -0,0 +1,57 @@
+-
+  var moment = UTILS.moment;
```
Comment (orlando) [inline apps/document-generation/src/templates/credit_report_dispute_letter/0.pug]: Should we remove these files and add them when we are actually porting the templates? @duranmla

File: apps/document-generation/.serverless/cloudformation-template-create-stack.json (hunk 1 above)
Comment (duranmla) [reply apps/document-generation/.serverless/cloudformation-template-create-stack.json]: Awesome. Will do.

File: apps/document-generation/src/templates/credit_report_dispute_letter/0.pug (hunk 2 above)
Comment (duranmla) [reply apps/document-generation/src/templates/credit_report_dispute_letter/0.pug]: Yes, I think you are right. BTW any ideas regarding the name of the template? Or makes sense to keep the 0, 1, 2...

**Comment by duranmla** [comment]
We're going to move this to its own repository at https://github.com/debtcollective/document-service

**Pull request closed without merging**